features/docs/cli/rerun_failing_tests.feature

```diff
@@ -0,0 +1,71 @@
+Feature:
+  Rerun gives you a way to get through flaky tests usually pass after a few runs.
+  This gives a development team a way forward other than disabling a valuable test.
+  
+  - Specify max rerun count in option
+  - Output information to the screen
+  - Output rerun information in test report
+  
+  Questions:
+  use a tag for flaky tests?  Global option to rerun any test that fails?
```

Comment:
If we're going to reproduce the Maven behaviour (which is what I suggest we start with) then we should just have a global option.